whitepaper.md

```diff
@@ -351,14 +351,15 @@ Implementing SLI, SLO and SLA metrics lets you measure service quality and custo
 
 Before we deep dive into the implementation details, we should get the definitions clear as they can be fairly confusing and sometimes be used interchangeably.
 
-* Service Level Indicator (SLI): An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.
-* Service Level Objective (SLO): An SLO is a service level objective: objective for how often you can afford for it to fail. a target value or range of values for a service level that is measured by an SLI
-* Service Level Agreement (SLA): a business contract that includes consequences of violating the SLO. This is a targeted percentage
+* Service Level Indicator (SLI): An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided. For example: Error Rate = total_requests / failed_requests
+* Service Level Objective (SLO): An SLO is a service level objective: objective for how often you can afford for it to fail. a target value or range of values for a service level that is measured by an SLI. For example: 99.99 % should be successful.
+* Service Level Agreement (SLA): a business contract that includes consequences of violating the SLO. This is a targeted percentage. For example: 99.9% of requests should be successful. Note this is less then SLO so the SLO is defended (via alerting) prior to the SLA potentially becoming impacted. 
```

Comment:
If we're adding examples of SLA/SLO and how SLO should be more relaxed than SLA, what do you think about reflecting that in the examples?

E.g. putting 95% for the SLO example and 99,99% for the SLA

Reply:
> Note this is less then SLO so the SLO is defended (via alerting) prior to the SLA potentially becoming impacted. 

The SLO % should be higher then the SLA % to alert of SLO breaches and to avoid SLA breaches?